README.md

```diff
@@ -4,40 +4,32 @@
 
 ---
 
-## TL;DR?
+## Welcome to Project Helix!
 
-* [Start Developing Your First Helix Project](https://main--helix-website--adobe.hlx.live/tutorial)
+* *NEW* Visit our [official website](https://www.hlx.live/)
```

Comment:
Need to say new? These things tend to get outdated.

Happy to send a PR in a few weeks, too. Not sure how many eyes are on this readme.